macaw-ppc/tests/PPC64Tests.hs

```diff
@@ -1,42 +1,55 @@
 {-# LANGUAGE DataKinds #-}
 {-# LANGUAGE DeriveDataTypeable #-}
 {-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE GADTs #-}
 {-# LANGUAGE GeneralizedNewtypeDeriving #-}
 {-# LANGUAGE RankNTypes #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TypeApplications #-}
 
 module PPC64Tests (
```

Comment:
Perhaps this should be renamed to just `PPCTests`, since this is no longer specific to 64-bit PPC.